CHANGELOG.md

```diff
@@ -20,9 +20,11 @@ security issues.
 
 ### IMPROVEMENTS
 
+- `[mempool]` Remove redundant error return in `(*Mempool).Update`
+  ([\#1288](https://github.com/cometbft/cometbft/pull/1288)
 - `[rpc]` Remove response data from response failure logs in order
-  to prevent large quantities of log data from being produced
-  ([\#654](https://github.com/cometbft/cometbft/issues/654))
+to prevent large quantities of log data from being produced
+([\#654](https://github.com/cometbft/cometbft/issues/654))
```

Comment:
We don't change the changelog manually. Please read [this section](https://github.com/cometbft/cometbft/blob/main/CONTRIBUTING.md#changelog) for instructions on how to proceed.